en/_posts/2019-12-07-rails-beginner-workshop.markdown

```diff
@@ -6,9 +6,11 @@ i18n_key: rails_beginner_workshop_2019_12
 
 ![Photo of dozen women with laptops, coding together around a large white table](/assets/images/Dec19-Banner.jpeg)
 
-code curious want to give absolute beginners the opportunity and community to learn the technologies needed to build their own website, with a hands on introduction to a web framework called Ruby on Rails. Attendees will be guided by coaches in building their website.
+### Let's dive into Coding <3
+code curious want to give absolute beginners the opportunity and community to learn the technologies needed to build their own website, with a hands on introduction to a web framework called Ruby on Rails. Attendees will be guided by coaches in building a website using our tutorials.
 
-We provide a unique learning environment to not only support, but also further motivate women who want to code, but don't have the right environment to do so. We are a community, so this is the start of a longer career changing path for many of our attendees.
+
```

Comment:
I suggest:

`[...] so this event often is the ignition point for a career change (even if it takes a while) for many [...]`